x-pack/plugins/cloud_integrations/cloud_links/public/maybe_add_cloud_links/appearance_selector/appearance_modal.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { type FC } from 'react';
+import {
+  EuiButton,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSpacer,
+  useGeneratedHtmlId,
+  EuiButtonEmpty,
+  EuiCallOut,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { IUiSettingsClient } from '@kbn/core-ui-settings-browser';
+import type { DarkModeValue as ColorMode } from '@kbn/user-profile-components';
+import { type Value, ValuesGroup } from './values_group';
+import { useAppearance } from './use_appearance_hook';
+
+const systemLabel = i18n.translate('xpack.cloudLinks.userMenuLinks.appearanceModalSystemLabel', {
+  defaultMessage: 'System',
+});
+
+const colorModeOptions: Array<Value<ColorMode>> = [
+  {
+    id: 'system',
+    label: systemLabel,
+    icon: 'desktop',
+  },
+  {
+    id: 'light',
+    label: i18n.translate('xpack.cloudLinks.userMenuLinks.appearanceModalLightLabel', {
+      defaultMessage: 'Light',
+    }),
+    icon: 'sun',
+  },
+  {
+    id: 'dark',
+    label: i18n.translate('xpack.cloudLinks.userMenuLinks.appearanceModalDarkLabel', {
+      defaultMessage: 'Dark',
+    }),
+    icon: 'moon',
+  },
+  {
+    id: 'space_default',
+    label: i18n.translate('xpack.cloudLinks.userMenuLinks.appearanceModalSpaceDefaultLabel', {
+      defaultMessage: 'Space default',
+    }),
+    icon: 'spaces',
+    betaBadgeLabel: i18n.translate('xpack.cloudLinks.userMenuLinks.appearanceModalBetaBadgeLabel', {
+      defaultMessage: 'Deprecated',
+    }),
+    betaBadgeTooltipContent: i18n.translate(
+      'xpack.cloudLinks.userMenuLinks.appearanceModalBetaBadgeTooltip',
+      {
+        defaultMessage: 'Space default settings will be deprecated in 10.0.',
```

Comment:
Are we sure we want to specifically call out 10.0 here, vs putting something less specific? While I would expect there are no concerns with removing in 10, it technically will still require approval

Reply:
@petrklapka Can you weigh in here? This came from Kate's design.

Reply:
I agree with Luke. I think we should always try to avoid specifying exact release numbers and dates (even if we're 1000% sure).